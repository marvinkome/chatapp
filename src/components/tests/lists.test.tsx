import React from 'react';
import renderer from 'react-test-renderer';

import Lists from '../lists';

describe('Lists tests', () => {
    it('renders correctly with defaults', () => {
        const app = renderer.create(<Lists renderItems={[]} />).toJSON();
        expect(app).toMatchSnapshot();
    });
});
