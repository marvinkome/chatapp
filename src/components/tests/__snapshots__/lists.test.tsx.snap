// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lists tests renders correctly with defaults 1`] = `
<RCTScrollView
  closeOnRowBeginSwipe={false}
  closeOnRowPress={true}
  closeOnScroll={true}
  dataArray={Array []}
  dataSource={
    ListViewDataSource {
      "items": 0,
    }
  }
  directionalDistanceChangeThreshold={2}
  disableLeftSwipe={false}
  disableRightSwipe={false}
  enableEmptySections={true}
  leftOpenValue={0}
  previewFirstRow={false}
  recalculateHiddenLayout={false}
  renderRow={[Function]}
  renderScrollComponent={[Function]}
  rightOpenValue={0}
  style={Object {}}
  swipeToOpenPercent={50}
>
  <View />
</RCTScrollView>
`;
